api/views.py

```diff
@@ -29,6 +29,37 @@ def index(request, name):
     field = request.GET.get('_source', '')
     sort = request.GET.get('sort', '')
     query = request.GET.get('q', '')
+    from_ = request.GET.get('from_', 0)
+    filters = request.GET.get('filters', '{}')      # Example: {field1: [val1, val2], field2: [val1, val2], ...}
+    aggregations = request.GET.get('aggs', '{}')    # Example: {aggName1: field1, aggName2: field2, ...}
+
+    # generate query for filtering
+    filter_values = []
+    not_filter_values = []
+    filters = json.loads(filters)
+    for key in filters.keys():
+        if (filters[key][0] != 'false'):
+            filter_values.append({"terms": {key: filters[key]}})
+        else:
+            not_filter_values.append({"terms": {key: ["true"]}})
+    filter_val = {}
+    if (len(filter_values)):
+        filter_val['must'] = filter_values
+    if (len(not_filter_values)):
+        filter_val['must_not'] = not_filter_values
+    if (len(filter_values) or len(not_filter_values)):
+        filters = {"query": {"bool": filter_val}}
```

Comment:
1. Remove redundant parentheses
2. Empty list will always be False in Python, so no need to check its length here 
3. Last if could be simplified to just "if filter_val:"